build-support/bin/download_binary.sh

```diff
@@ -4,6 +4,11 @@
 # Uses a reimplementation of the BinaryUtils mechanism.
 # Outputs an absolute path to the binary, whether fetched or already present, to stdout.
 
+# Use this version as a fallback for platform-specific binaries to download if we are on a version
+# of OSX that we don't explicitly have binaries for yet.
+# TODO: Keep this up to date automatically somehow!
```

Comment:
Should probably link to a GitHub issue here

Reply:
Yes! This is obviated with the new python approach but this is correct.